ios/Video/RCTVideo.swift

```diff
@@ -611,10 +627,86 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     }
 
+    func getAudioTrackInfo(
+        model _: M3U8PlaylistModel,
+        principalModel: M3U8PlaylistModel
+    ) -> [String: Any] {
+        var streamList: NSArray = .init()
+
+        for i in 0 ..< principalModel.masterPlaylist.xStreamList.count {
+            let inf = principalModel.masterPlaylist.xStreamList.xStreamInf(at: i)
+            if let inf {
+                streamList.adding(inf)
+            }
+        }
+
+        if let currentEvent: AVPlayerItemAccessLogEvent = _player?.currentItem?.accessLog()?.events.last {
+            let predicate = NSPredicate(format: "%K == %f", "bandwidth", currentEvent.indicatedBitrate)
+            let filteredArray = streamList.filtered(using: predicate)
+            let current = filteredArray.last
+
+            if let current = current as? M3U8ExtXStreamInf {
+                let mediaList: NSArray = .init()
+                for i in 0 ..< principalModel.masterPlaylist.xMediaList.audio().count {
+                    let inf = principalModel.masterPlaylist.xMediaList.audio().xMedia(at: i)
+                    if let inf {
+                        mediaList.adding(inf)
+                    }
+                }
+
+                let predicate = NSPredicate(format: "SELF.groupId == %@", current.audio)
+                let currentAudio = mediaList.filtered(using: predicate).last
+
+                if let currentAudio = currentAudio as? M3U8ExtXMedia {
+                    let url: URL = currentAudio.m3u8URL()
+                    let audioModel: M3U8PlaylistModel? = try? .init(url: url)
+
+                    if let audioModel {
+                        let audioInfo: M3U8SegmentInfo = audioModel.mainMediaPl.segmentList.segmentInfo(at: 0)
+
+                        return [
+                            "title": currentAudio.name(),
+                            "language": currentAudio.language(),
+                            "codecs": currentAudio.groupId(),
+                            "file": audioInfo.uri.absoluteString,
+                            "channels": currentAudio.channels,
+                        ]
+                    }
+                }
+            }
+        }
+
+        return .init()
+    }
+
+    func getVideoTrackInfo(
+        model: M3U8PlaylistModel,
+        principalModel: M3U8PlaylistModel
+    ) -> [String: Any] {
+        // swiftlint:disable:next empty_count
+        if !(model.mainMediaPl.segmentList.count == 0) {
+            let uri: URL = model.mainMediaPl.segmentList.segmentInfo(at: 0).uri
+
+            var codecs = ""
+            // swiftlint:disable:next empty_count
+            if !(principalModel.masterPlaylist.xStreamList.count == 0) {
+                if let inf = principalModel.masterPlaylist.xStreamList.xStreamInf(at: 0) {
+                    codecs = (inf.codecs as NSArray).componentsJoined(by: ",")
+                }
+            }
+
+            let stringURL = uri.absoluteString as NSString
+            return [
```

Comment:
trzeba przejrzeć się temu co jest w dokumentacji https://miquido.github.io/react-native-video/component/events#onvideotracks kontra temu co jest zwracane tutaj. Bo to nie do końca wygląda na spójne. 

Czy możemy tutaj dodać coś więcej co jest na androidzie?

Reply:
Widzę zmienę w docs/pages/component/events.mdx tylko o tym że wspierane na iOS.

Obecnie tabelka wygląda tak https://miquido.github.io/react-native-video/component/events#onvideotracks

A ma być chyba tak

Property | Type | Description | Zmiana
-- | -- | -- | --
index | number | Track index | dodać (android only), ale może uda się to jakoś dodać na ios 
trackId | string | Track ID of the video track | dodać (android only)
codecs | string | MimeType of codec used for this track
width | number | Track width | dodać (android only), ale może uda się to jakoś dodać na ios 
height | number | Track height | dodać (android only), ale może uda się to jakoś dodać na ios 
bitrate | number | Bitrate in bps | dodać (android only)
selected | boolean | true if track is selected for playing | nie jest zaimplementowane i trzeba to jakoś dodać
file | string | File name from DASH manifest | zmienić opis from DASH manifest on Android oraz from M3U8 file on iOS
supplementalProperties | string | Supplemental properties for the adaptation set from DASH manifest | dodać (android only)